File: OneCircleWeb/scripts/Shop.js
define(["Place"], function(Place){
    "use strict";

    function Shop(title, latitude, longitude, whatDoTheySell, openHours) {
        Place.apply(this, arguments);
        this.whatDoTheySell = whatDoTheySell;
        this.openHours = openHours;
    }
    Shop.prototype = Object.create(Place.prototype);
    
    return Shop;    
});

File: OneCircleWeb/DAL/Shop.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace OneCircleWeb.DAL
{
    public class Shop : Place
    {
        public string  WhatDoTheySell { get; set; }
        public string OpenHours { get; set; }
    }
}